src/Test.UnitTests.Sarif.Driver/Sdk/ExportConfigurationCommandBaseTests.cs

```diff
@@ -76,12 +74,20 @@ public void ExportConfigurationCommandBase_VerifyAllRulesDisabledConfiguration()
             PropertiesDictionary configuration = CreateAllRulesDisabledConfiguration();
 
             configuration.Should().NotBeNull();
-            configuration.Keys.Count.Should().Be(3);
+
+            // We export two rules, FunctionlessTestRule and TestRule
+            configuration.Keys.Count.Should().Be(NumberOfExportedRules());
 
             foreach (PropertiesDictionary ruleProperties in configuration.Values)
             {
                 ((RuleEnabledState)ruleProperties[DefaultDriverOptions.RuleEnabled.Name]).Should().Be(RuleEnabledState.Disabled);
             }
         }
+
+        private static int NumberOfExportedRules()
+        {
+            return typeof(ExportConfigurationCommandBaseTests).Assembly.GetTypes()
+                .Count(t => t.GetCustomAttributes(typeof(ExportAttribute)).Count() > 0);
```

Comment:
>ExportAttribute [](start = 57, length = 15)

Potential bug: You could miscount if there are type which export anything other than `typeof(ReportingDescriptor)`.